### PR TITLE
remove stale polkadot call filter

### PR DESCRIPTION
**Pull request by kianenigma**
The call filter was put in place in the early POA days and is no longer needed.

### CODE REVIEW

**Review by ggwpez — APPROVED** [review]

**Review by rphmeier — APPROVED** [review]

**Comment by rphmeier** [comment]
bot merge

File: runtime/kusama/src/lib.rs [hunk 1]
```diff
@@ -163,7 +155,7 @@ parameter_types! {
 }
 
 impl frame_system::Config for Runtime {
-	type BaseCallFilter = BaseFilter;
+	type BaseCallFilter = runtime_common::AllowAllCalls;
```
Comment (bkchr) [inline runtime/kusama/src/lib.rs:158]: Why not
```suggestion
	type BaseCallFilter = frame_support::traits::Everything;
```
?

Comment (kianenigma) [reply runtime/kusama/src/lib.rs:158]: voila, I didn't know this existed.

**Review by bkchr — APPROVED** [review]

**Comment by kianenigma** [comment]
bot merge

**Comment by kianenigma** [comment]
bot rebase

**Comment by kianenigma** [comment]
bot merge

File: runtime/polkadot/src/lib.rs [hunk 2]
```diff
@@ -135,62 +132,6 @@ pub fn native_version() -> NativeVersion {
...
-			Call::Hrmp(_) |
-			Call::Slots(_) |
-			Call::Registrar(_) |
-			Call::Auctions(_) |
-			Call::Crowdloan(_) |
-			Call::VoterList(_) |
-			Call::XcmPallet(_) |
-			Call::NominationPools(_) => true,
-			// All pallets are allowed, but exhaustive match is defensive
-			// in the case of adding new pallets.
```
Comment (shawntabrizi) [inline runtime/polkadot/src/lib.rs:189]: i dont disagree with this PR, but this was stated to be explicit

**Pull request merged**